### PR TITLE
Styled improvment

### PR DESCRIPTION
**Pull request by Gosia-Magdzik**
Hi there!
I improved our vebsite on styled site. Now it looks much better as well as on big screen as on mobile devices;

### CODE REVIEW

File: src/common/Nav/index.js [hunk 1]
```diff
@@ -36,8 +40,16 @@ export const Nav = () => (
...
           <Magnifier src={magnifier} />
-          <InputStyled placeholder={`Search for movies..`} />
+          <InputStyled 
+            placeholder={
+              `Search for movies..
+              ${location.pathname.includes("movies" 
+                      ? "movies..." 
+                      : "people...")
+              }`} 
+          />
```
Comment (MarcinkPawel) [inline src/common/Nav/index.js:50]: Nice work, but I think it should be ` `Search` for `

File: src/common/Pagination/styled.js [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
     justify-content: center;
-    align-items: center;
-    gap: 8px;
-    margin-bottom: 123px;
-    margin-top: 40px;
+    gap: 12px;
+    margin: 40px 0 103px;
+    position: static;
 
     @media (max-width: ${({ theme }) => theme.breakpoint.MobileMax}px) {
```
Comment (MarcinkPawel) [inline src/common/Pagination/styled.js:11]: it mobileMax, not MobileMax. It starts with lower case letter.

File: src/features/movies/moviesPage/BackgroundPoster/index.js [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
-        <Title>Mulan</Title>
+        <Title>Mulan long Title</Title>
         <Rating>
           <Star src={star} />
-          <Rate>3/10</Rate>
-          <Votes> 311 votes</Votes>
+          <RateStyledA>3</RateStyledA>
+          <RateStyledB>/ 10</RateStyledB>
         </Rating>
+          <Votes> 311 votes</Votes>
```
Comment (MarcinkPawel) [inline src/features/movies/moviesPage/BackgroundPoster/index.js:18]: I think we need to treat Rate numer as variables, then RateStyledB should be jus "10" and "/" as <p>. The same with "311 votes", "311" use as variable and "votes" as <p>.

File: src/features/movies/moviesPage/index.js [hunk 4]
```diff
@@ -1,17 +1,12 @@
...
 export const Movie = () => (
   <>
-  <Wrapper>
     <BackgroundPoster />
     <AboutMovie />
-  </Wrapper>
     <CrewCast />
-    <Pagination/>
-    </>
+  </>
```
Comment (MarcinkPawel) [inline src/features/movies/moviesPage/index.js:11]: Here component CrewCast should be rendered twice, first is for Cast, second for Crew.

File: src/common/Pagination/styled.js (hunk 2 above)
Comment (Gosia-Magdzik) [reply src/common/Pagination/styled.js:11]: Yes, of course! You are right

File: src/common/Nav/index.js (hunk 1 above)
Comment (Gosia-Magdzik) [reply src/common/Nav/index.js:50]: Yes! of course you are right

**Comment by Gosia-Magdzik** [comment]
Ready to merge

**Review by MarcinkPawel — APPROVED** [review]
Nice!😄

**Pull request merged**